rewrite-java-test/src/test/java/org/openrewrite/java/format/MinimumViableSpacingTest.java

```diff
@@ -138,10 +158,10 @@ class Test {
                   void foo(final int[] arr) {
                       for (int n = 0, x = 0; n < 100; n++, x++) {
                       }
-
+              
```

Comment:
This is sometimes annoying about IntelliJ; a quick change on a different line followed by auto format should clear these out.